main.go

```diff
@@ -104,6 +107,18 @@ func main() {
 		log.Fatalln("Failed to start workers", err)
 	}
 
+	hcb := healthcheck.BuildHealthCheckBackend(postgresScoopConnection, postgresBackend)
```

Comment:
I dislike the naming convention hcb, hch, and healthServeMux.

I'd suggest:
hcBackend
hcHandler
hcServeMux